osquery/utils/aws_util.cpp

```diff
@@ -219,7 +221,7 @@ OsquerySTSAWSCredentialsProvider::GetAWSCredentials() {
       initAwsSdk();
     }
 
-    makeAWSClient<Aws::STS::STSClient>(client_, false);
+    makeAWSClient<Aws::STS::STSClient>(client_, "", false);
```

Comment:
Oh, this wasn't my code. I ajust dded additional argument. But since I touched it, I will fix it :+1: